tests/test_github.py

```diff
@@ -0,0 +1,101 @@
+# This file is part of the github2jira project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+#
+
+import os
+
+from unittest import mock
+import responses
+
+import github2jira.githublib as githublib
+from tests.common import get_raw_issue
+
+
+def mockenv(**envvars):
+    return mock.patch.dict(os.environ, envvars)
+
+
+def test_issue_properties():
+    raw_issue = get_raw_issue()
+    issue = githublib.Issue(raw_issue)
+
+    assert issue.repo == "repo"
```

Comment:
Please ref the expected result from the place it is defined, do not repeat it yourself here as it is unclear why this is the value and not something else.

In some places you did it right.